### PR TITLE
Add separate emergency shutdown for main smart contracts

### PR DESCRIPTION
**Pull request by philipdesmedt**

### CODE REVIEW

File: clarity/contracts/dao.clar [hunk 1]
```diff
@@ -67,18 +67,14 @@
   )
 )
 
-(define-public (trigger-emergency-shutdown)
+(define-public (toggle-emergency-shutdown)
```
Comment (nieldeckx) [inline clarity/contracts/dao.clar:70]: Q: Is this still needed? Only freddie is using this, but that contract has it's own shutdown function.

**Review by nieldeckx — COMMENTED** [review]
Looks good! A couple of tests to prove this is working would be nice :)

File: clarity/contracts/dao.clar (hunk 1 above)
Comment (philipdesmedt) [reply clarity/contracts/dao.clar:70]: All contracts are using this. This way you have the option of disabling 1 contract or the whole system by the DAO.

**Comment by philipdesmedt** [comment]
@nieldeckx 👍 thx - will write tests for all of the shutdown triggers

**Pull request merged**